docs/user/exams/instructors_guide.rst

```diff
@@ -207,12 +207,23 @@ During the exam creation and configuration, you can create your exam and configu
 - To register students to the exam, navigate from the exam management to the *Students* page. Artemis offers two options to register students. You can:
 
     1. Add students manually by searching via the search bar.
-    2. Bulk import students using a ``CSV`` file. You can do this by pressing the |import_students| button.
+    2. Bulk import students using a ``CSV`` file. You can do this by pressing the |import_students| button. An exemplatory file can be found here: :download:`csv <instructor/example_import_students.csv>`.
     3. Register every student in the course. You can do this by pressing the |register_course_students| button.
 
 .. note::
     Just registering the students to the exam will not allow them to participate in the exam. First, individual :ref:`student exams <student_exams>` must be generated.
 
+.. note::
+    Seats and rooms can currenlty only be assigned via the bulk import of students, by defining the seat and room in the ``CSV`` file.
+
+.. note::
+    For more information on the Exam Participation Checker iPad application have a look here: :ref:`Exam Participation Checker <exam_participation_checker>`.
+
+- For using the Exam Participation Checker iPad app it is necessary to upload images for the students. This can be done via the |upload_images| button. An exemplatory file can be found here: :download:`pdf <instructor/example_upload_images.pdf>`.
+
+.. note::
+    To upload the images each student needs to have a matriculation number assigned. This is used to match the image with the according student.
```

Comment:
This whole section is only really necessary when using the app. Therefore, I’d replace this new section with a note something like:

```rst
..note::
    Artemis also supports validating participants signatures for in-person exams.
    For more information please see :ref:`Exam Participation Checker <exam_participation_checker>`.
```

The remaining information I’d move into a new ‘Requirements’ section on the app documentation page. That way all information necessary to use the app (except base-exam setup, which is already linked to) is available on a single page.